lib/core/src/Cardano/Wallet/Primitive/Types/Tx/TxSeq.hs

```diff
@@ -0,0 +1,544 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- |
+-- Copyright: © 2022 IOHK
+-- License: Apache-2.0
+--
+-- This module provides the 'TxSeq' type and related functions.
+--
+-- The 'TxSeq' type provides a way to model a linear sequence of transactions
+-- and intermediate UTxO sets, where each individual transaction 't_i_j'' is
```

Comment:
```suggestion
-- and intermediate UTxO sets, where each individual transaction 't_i_j' is
```

Reply:
You clearly have the eyes of an eagle! 🦅

Thanks!

Reply:
Fixed in f36e0a45fea20f3e64b09ba26cdf76b8a66d4fd2.